Pipfile

```diff
@@ -15,7 +15,7 @@ dateparser = "*"
 mlflow = "*"
 numpy = "*"
 scikit-learn = "*"
-tensorflow = "*"
+tensorflow = ">=2.0, <2.5"
```

Comment:
@chauhankaranraj can you make the same changes to [requirements.txt](https://github.com/AICoE/prometheus-anomaly-detector/blob/master/requirements.txt#L11) and the [environments.yml](https://github.com/AICoE/prometheus-anomaly-detector/blob/master/environment.yml#L18) as well?

Reply:
Ofc, updated!